lib/commands/screenshots.js

```diff
@@ -4,39 +4,28 @@ import { getScreenshot as simctlGetScreenshot } from 'node-simctl';
 import { exec } from 'teen_process';
 import log from '../logger';
 import { fs, tempDir, util, imageUtil } from 'appium-support';
+import jimp from 'jimp';
 
 let commands = {};
 
 async function getScreenshotWithIdevicelib (udid, isLandscape) {
-  const pathToScreenshotTiff = await tempDir.path({prefix: `screenshot-${udid}`, suffix: '.tiff'});
-  await fs.rimraf(pathToScreenshotTiff);
   const pathToResultPng = await tempDir.path({prefix: `screenshot-${udid}`, suffix: '.png'});
   await fs.rimraf(pathToResultPng);
   try {
     try {
-      await exec('idevicescreenshot', ['-u', udid, pathToScreenshotTiff]);
+      await exec('idevicescreenshot', ['-u', udid, pathToResultPng]);
```

Comment:
good, that it was fixed

Reply:
Yeah. For a long time idevicescreenshot still created files with .tiff extensions even though they were actually PNGs